src/scripts/setup_test_data/setup_test_data.py

```diff
@@ -98,6 +98,7 @@ def add_test_data(syn, dir_path, bucket_name, folder_id):
           with archive.open('metadata.json') as metadata_file:
             metadata = json.load(metadata_file)
             file_metadata['taskIdentifier'] = metadata['taskIdentifier']
+            file_metadata['appversion'] = metadata['appVersion']
```

Comment:
This script, which sets up the testing data, had to be updated for changes in S3ToJson Glue script.